sklearn/datasets/samples_generator.py

```diff
@@ -613,20 +613,23 @@ def make_circles(n_samples=100, shuffle=True, noise=None, random_state=None,
 
     if factor > 1 or factor < 0:
         raise ValueError("'factor' has to be between 0 and 1.")
+        
```

Comment:
./sklearn/datasets/samples_generator.py:616:1: W293 blank line contains whitespace

Reply:
Thanks. 
It seems PyCharm is not the solution to every PEP8-related issue. :/